cfgov/jinja2/v1/_includes/organisms/mega-menu-var-1.html

```diff
@@ -62,7 +62,7 @@
             {% if nav_depth == 1 -%}
             <li class="m-list_item {{ _classes( nav_depth, '-item' ) }}">
                 {% import 'molecules/global-header-cta.html' as global_header_cta with context %}
-                {{ global_header_cta.render() }}
+                {{ global_header_cta.render( language=language ) }}
```

Comment:
I'm surprised this works (here and in the other menu template), because `language` isn't a parameter to the `_nav_list` macro. This might be because the whole file is imported [`with context`](https://github.com/cfpb/cfgov-refresh/blob/f31b1c31439aae237744a2805a3c812218fde201/cfgov/jinja2/v1/_layouts/base.html#L252) which means that `language` is available globally. I hate to be nitpicky, but maybe to be consistent we should explicitly pass it to `_nav_list` as well so that this works even without context?

Reply:
Aw yeah it should probably mirror how `_nav_level` does it in the same file. 👍